process/project-lifecycle.md

```diff
@@ -0,0 +1,178 @@
+
+#### Incubation Entry Requirements and Considerations
+
+* Projects must have a minimum of three maintainers with a minimum of two different company affiliations.
+* Projects should be able to show adoption by multiple parties and adoption's value to the open source community and/or end users (may include adoption of beta/early versions).
+* Projects must be aligned with the OpenSSF mission and either be a novel approach for existing areas, address an unfulfilled need, or be initial code needed for OpenSSF WG work. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
+* Projects must have documented, initial project governance
+
+#### Project Process: Sandbox to Incubation and direct entry to Incubation
+
+Projects are required to undergo technical due diligence as a part of the process of entering Incubation, whether as a move from Sandbox to Incubation or enterting Incubation directly. Technical Due Diligence is driven by a TAC or parent WG sponsor with close collaboration of the project. Once the diligence is complete and the proposal made, the Due Diligence document is made available to the community for two weeks to solicit public comment before a TAC vote is called.
+
+
+
+### Graduated
+
+Graduated projects signal the highest level of maturity for an OpenSSF project. Graduated projects have consistent, mature open source development practices with sustained contribution and activity. 
+
+#### Project Reponsibilities
+
+* Bi-annual updates to the TAC on technical vision and progress on vision
+* Maintain a diversified contributor base (i.e. not a single-vendor project) with an active flow of contributions
+* Follow security best practices, including achieving a [Silver OpenSSF Best Practices badge](https://bestpractices.coreinfrastructure.org/en/criteria)
+* Maintain a point of contact for vulnerability reports and follow coordinated vulnerability disclosure practices
+* Implement, practice, and refine mature software development and release practices, such as adherence to semantic versioning, and having a declared policy for stable releases and backported fixes.
+
+#### Project Support
+* Receives guidance on technical direction from TAC 
+* Receives space for Contributor Summits at OpenSSF bi-annual events
+* Receives OpenSSF Code of Conduct Committee support
+* Receives infrastructure support (details determined by project leads and OpenSSF Budget Committee)
+* Receives support with vulnerability disclosure from the OpenSSF (Vulnerability Disclosure WG)
+* May post project updates and tutorials to the OpenSSF blog
+* May request OpenSSF budget for project improvements such as security audits or time-bound contracting needs
+* May request OpenSSF for sustained maintainer stipends (details determined by OpenSSF and project leads)
+* With additional TAC or WG approval, may fundraise for dedicated project funds, coordinated by the OpenSSF
```

Comment:
same comment as Incubating:

Does this statement line up to the Project w/ SIF model listed in `organizational-structure-overview.md`? If so, I don't think WG approval to create a SIF is sufficient.